zero/src/bin/trie_diff.rs

```diff
@@ -0,0 +1,159 @@
+use std::io::Read;
+use std::iter::repeat;
+use std::path::PathBuf;
+use std::sync::Arc;
+
+use anyhow::Result;
+use clap::{Parser, ValueHint};
+use evm_arithmetization::generation::DebugOutputTries;
+use futures::{future, TryStreamExt};
+use paladin::directive::{Directive, IndexedStream};
+use paladin::runtime::Runtime;
+use regex::Regex;
+use trace_decoder::observer::TriesObserver;
+use tracing::{error, info};
+use zero::ops::register;
+use zero::prover::{cli::CliProverConfig, BlockProverInput, ProverConfig};
+
+#[derive(Parser)]
+#[command(version = zero::version(), propagate_version = true)]
+pub(crate) struct Cli {
```

Comment:
A high level description here would really help me to understand why someone might use this, and what they'd want from it